kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -251,7 +251,7 @@ class ResidualBasedBlockBuilderAndSolver
 
 
             //#pragma omp parallel for firstprivate(nconditions, LHS_Contribution, RHS_Contribution, EquationId ) schedule(dynamic, 1024)
-            #pragma omp parallel for  schedule(guided, 512)
+            #pragma omp for  schedule(guided, 512)

```

Comment:
the question is not "naive" :-)

according to 
https://software.intel.com/en-us/articles/openmp-loop-scheduling

the 512 is the minimum chunk size. It means it will not go below chunks of 512.
We could put a different number, i did not test it, i saw that it being used like this in an other contest